sdk/Lusid.Sdk.Tests/tutorials/Instruments/DemoEquityOption.cs

```diff
@@ -0,0 +1,181 @@
+
+            // create Black-Scholes recipe specifying where to look for market data and which metrics to return
+            // if in a larger portfolio, we would make a specific VendorModelRule specifying that equity options are to be valued using Black-Scholes
+            var recipeCode = "EquityOption_ValuationRecipe";
+            var pricingOptions = new PricingOptions(new ModelSelection(ModelSelection.LibraryEnum.Lusid, ModelSelection.ModelEnum.BlackScholes));
+            var recipe = new ConfigurationRecipe
+            (
+                scope: scope,
+                code: recipeCode,
+                market: new MarketContext(
+                    new List<MarketDataKeyRule> {},
+                    options: new MarketOptions(defaultSupplier: "Lusid", defaultScope: scope, defaultInstrumentCodeType: "RIC")
+                    ),
+                pricing: new PricingContext(options: pricingOptions)
+            );
+            _testDataUtilities.UpsertRecipe(recipe);
+
+            // define the metrics that we wish to return
+            string ValuationDateKey = "Analytic/default/ValuationDate";
+            string InstrumentTag = "Analytic/default/InstrumentTag";
+            string HoldingPvKey = "Holding/default/PV";
+            var valuationSpec = new List<AggregateSpec>
+            {
+                new AggregateSpec(ValuationDateKey, AggregateSpec.OpEnum.Value),
+                new AggregateSpec(InstrumentTag, AggregateSpec.OpEnum.Value),
+                new AggregateSpec(HoldingPvKey, AggregateSpec.OpEnum.Value)
+            };
+
+            // choose valuation dates
+            var valuationSchedule = new ValuationSchedule(effectiveAt: _demoEffectiveAt);
+
+            // construct and perform valuation request
+            var instruments = new List<WeightedInstrument> {new WeightedInstrument(1, "some-holding-identifier", option)};
+            var inlineValuationRequest = new InlineValuationRequest(
+                recipeId: new ResourceId(scope, recipeCode),
+                metrics: valuationSpec,
+                sort: new List<OrderBySpec> {new OrderBySpec(ValuationDateKey, OrderBySpec.SortOrderEnum.Ascending)},
+                valuationSchedule: valuationSchedule,
+                instruments: instruments);
+            var valuation = _aggregationApi.GetValuationOfWeightedInstruments(inlineValuationRequest);
+            Assert.That(valuation, Is.Not.Null);
+            Assert.That(valuation.Data.Count, Is.EqualTo(instruments.Count));
+
+            var pv = valuation.Data[0][HoldingPvKey];
+            Assert.That(pv, Is.Positive);
+            Console.WriteLine($"Computed pv of {pv} at time {_demoEffectiveAt:O}");
+        }
+
+        [Test]
+        public void DemoEquityOptionCashFlows()
+        {
+            var scope = "DemoEquityOptionCashFlows";
+
+            // upsert option
+            var option = InstrumentExamples.CreateExampleEquityOption();
```

Comment:
You meant // CREATE an equity option (that can then be upserted into Lusid) (first)?

Reply:
Technically yes, changed, although I was referring to the behaviour of the code block, not the line.